### DISCUSSION TITLE
Antivirus block me to enter Mozilla Hubs

### DISCUSSION THREAD

**Discussion by cwh690** [mozilla/hubs#4150] [General Questions & Discussion]
When I want to enter the Mozilla Hubs, my antivirus program blocks me. Is it got another way to enter without closing my antivirus program?

**Comment by rawnsley** [comment]
Can you provide some more information? Name and version of the antivirus, how far you can get, what the page looks like when it fails to load, [developer console logs](https://zapier.com/help/troubleshoot/behavior/view-and-save-your-browser-console-logs), etc...

You should look first at your antivirus program and how to add exceptions to the rules it is applying. For instance, add hubs.mozilla.com to the domain whitelist.

**Reply by cwh690** [reply]
Hi, I am using Avast Antivirus, it shows me like this.
![avast](https://user-images.githubusercontent.com/82493677/114688840-afa50b00-9d47-11eb-88b4-16b530702d65.jpg)

**Reply by rawnsley** [reply]
I suggest you file it with Avast as a false positive: https://www.avast.com/false-positive-file-form.php

Try and include as much information as possible including an example web link.

**Reply by clementabai** [reply]
@ rawnsley... thanks! I had the same exact problem and added the "https://hubs-proxy.com/*" url to Avast "exception" list under "settings" and that took care of it.
